<?xml version="1.0"?>
<launch>
 <node pkg="pylon_camera" name="sol_camera" type="pylon_camera_node" output="screen">
  <param name="camera_frame" value="/sol_camera" type="string" /> <!-- frame to which the camera is attached -->
  <param name="start_exposure" value="2000" type="int" />
  <param name="use_sequencer" value="false" type="bool" />
  <param name="desired_framerate" value="10.0" type="double" />
  <param name="use_brightness" value="true" type="bool" />
  <param name="magazino_cam_id"  value="TORU_0002_SOL"  type="string" />
  <param name="intrinsic_yaml_string" command="cat $(find pylon_camera)/calib/TORU_0002_SOL.yml"/>
 </node>
 <node name="image_view" pkg="image_view" type="image_view" respawn="false" output="screen">
  <remap from="image" to="/sol_camera/image_rect"/>
 </node>
</launch>
